content/general/concepts/virtual-machines/virtual-machines.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: 'Virtual Machines'
+Description: 'Virtual machines are virtual versions of physical computers.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Virtual Machines'
+  - 'Cloud Computing'
+  - 'Security'
+  - 'DevOps'
```

Comment:
Let's include any tags that come from the Docs [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) file. What I'd suggest is the following:

```suggestion
  - 'Servers'
  - 'Cybersecurity'
```

Though I want to check with my colleagues and see if we can at least add a `'Virtualization'` tag 😄